src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -47,6 +47,8 @@ const ITEM_TYPES = {
   PICKER: 'picker'
 };
 
+const IMAGE_TYPE_REGEX = /\.(gif|jpe?g|tiff?|png|webp|bmp)$/i;
```

Comment:
seems to work fine

![Bildschirmfoto 2022-05-18 um 13 52 53](https://user-images.githubusercontent.com/1942953/169033472-4e48893f-24e8-4c47-a9e3-653c1194397f.png)